File: recaudacion/2.Codigo/backend/RecaudacionApiLiquidacion/Domain/ClasificadorIngreso.cs
namespace RecaudacionApiLiquidacion.Domain
{
    public class ClasificadorIngreso
    {
        public int ClasificadorIngresoId { get; set; }
        public string Codigo { get; set; }
        public string Descripcion { get; set; }
        public int CuentaContableIdDebe { get; set; }
        public int CuentaContableIdHaber { get; set; }
        public bool Estado { get; set; }
    }
}

File: recaudacion/2.Codigo/frontend/src/app/core/models/clasificadoringreso.ts
export class ClasificadorIngreso {
    clasificadorIngresoId: number;
    codigo: string;
    descripcion: string;
    cuentaContableIdDebe: number;
    cuentaContableIdHaber: number;
    tipoTransaccion: number;
    generica: number;
    subGenerica: number;
    subGenericaDetalle: number;
    especifica: number;
    especificaDetalle: number;
    estado: boolean;
    usuarioCreador: string;
    usuarioModificador: string;
    index:number;
}

export class ClasificadorIngresoFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn:string;
    sortOrder: string;
    codigo:string;
    descripcion:string;
    estado:boolean;
}
